wiki/osu!_wiki/en.md

```diff
@@ -31,7 +31,7 @@ Articles are composed according to the [styling criteria](/wiki/Article_styling_
 
 Despite having the *wiki* suffix in its name, the osu! wiki does not fully satisfy [the Wiki concept](https://en.wikipedia.org/wiki/Wiki):
 
-- Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is expected, which raises the threshold necessary for entry.
+- Users are encouraged to improve existing articles and create new ones, but basic knowledge of [GitHub](https://github.com) is necessary in order to contribute, which raises the threshold necessary for entry. The osu! wiki does however provide some quick guides in the [contribution guide](/wiki/osu!_wiki/Contribution_guide) in order to alleviate this issue.
```

Comment:
"... provides _guides_ in the contribution _guide_ ..."

Reply:
was thinking "tutorials" but thought that was a bit of a clunkier notion to what are more like quick-start guides than actual github tutorials